test/distributed/fsdp/test_fsdp_param_exec_order_policy.py

```diff
@@ -257,17 +273,25 @@ def test_train_with_full_state_dict(
     def test_fsdp_flatten_params_exec_order(self, iters: int):
         model = CNN().cuda()
         group = dist.distributed_c10d._get_default_group()
-        fsdp_model = FSDP(model, group, auto_wrap_policy=ParamExecOrderPolicy(HandleInitMode.MODULE_LEVEL))
+        # Used to test non-default module_level_group_policy
+        module_level_policy = functools.partial(
+            transformer_auto_wrap_policy,
+            transformer_layer_cls={CNN, DoubleConv, nn.Linear},
```

Comment:
Just as an FYI related to this: https://github.com/pytorch/pytorch/issues/81050

Reply:
yes totally agreed, this should not be called transformer policy lol